spec/hashie/mash_spec.rb

```diff
@@ -977,14 +989,6 @@ class SubMash < Hashie::Mash
         expect(subject.dig('a', 'b')).to eq(1)
       end
 
-      context 'with numeric key' do
```

Comment:
@dblock do you think this removal should be added to the changelog under `Removed`? I don't actually consider this removal a bug fix as I think you mentioned...I believe it was intentionally specced behavior. I can't speak to _why_ anyone would rely on indifference dig access for numeric and string keys, but in the case anyone is, I think it should be noted. I'll add it and lmk if you disagree.

Reply:
No need, but belongs in UPGRADING.